src/streams/registry.h

```diff
@@ -4,14 +4,16 @@
  * This file is part of libgit2, distributed under the GNU GPL v2 with
  * a Linking Exception. For full terms see the included COPYING file.
  */
-#ifndef INCLUDE_streams_curl_h__
-#define INCLUDE_streams_curl_h__
+#ifndef INCLUDE_streams_registry_h__
+#define INCLUDE_streams_registry_h__
 
 #include "common.h"
-
 #include "git2/sys/stream.h"
 
-extern int git_curl_stream_global_init(void);
-extern int git_curl_stream_new(git_stream **out, const char *host, const char *port);
+/** Configure stream registry. */
+int git_stream_registry_global_init(void);
+
+/** Lookup a stream registration. */
+extern int git_stream_registry_lookup(git_stream_registration *out, int tls);
```

Comment:
There's a type mismatch here. `tls` is no integer but an enum now

Reply:
Oops.  Thanks, fixed.